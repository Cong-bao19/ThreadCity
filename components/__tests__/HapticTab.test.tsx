import React from 'react';
import { create } from 'react-test-renderer';
import { HapticTab } from '../HapticTab';

describe('HapticTab', () => {
    it('renders correctly', () => {
        const props = { onPressIn: jest.fn(), children: 'Test' };
        const tree = create(<HapticTab {...props} />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
