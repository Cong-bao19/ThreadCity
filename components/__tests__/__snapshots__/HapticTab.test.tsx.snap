// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HapticTab renders correctly 1`] = `
<View
  onPressIn={[Function]}
>
  Test
</View>
`;
